statsmodels/base/model.py

```diff
@@ -745,7 +746,7 @@ def fit(self, start_params=None, method='nm', maxiter=500, full_output=1,
     #fit.__doc__ += LikelihoodModel.fit.__doc__
 
 
-class Results(object):
+class Results(dimensions.NEQsMixin, dimensions.NobsMixin):
```

Comment:
You're right.  As mentioned elsewhere, for the purposes of keeping this example small I didn't port the necessary retro-fitting.

GenericLikelihoodModelResults attaches `model.endog`, `model.exog`, and sets `nobs`.  `LikelihoodModelResults` does not.  My preference would be to consistently set commonly-needed attributes in `LikelihoodModelResults` (or even `Results`), but there are any number of ways to approach this.  Ideally I'd want to avoid pinning data-like objects to results.

The point is that these are all commonly-needed attributes and we should never have to guess whether or not they exist and/or have been initialized.